lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
So why does the lookup plugin allow for defining a default, instead of using the built in default filter?

That would make it more consistent with our playbooks, more modular, and would allow us to use the features of the default filter (like replacing an empty string, empty list, etc with the default value if we want to).

For example:
```
msg: "{{ lookup('oo_option', '<key>') | default('<default_value>') }}"
```